framework/src/modules/token/commands/transfer.ts

```diff
@@ -43,6 +48,42 @@ export class TransferCommand extends BaseCommand {
 
 		try {
 			validator.validate(transferParamsSchema, params);
+
+			const senderAddress = utils.hash(context.transaction.senderAddress);
+			const tokenID = context.transaction.senderAddress;
+			const userStore = this.stores.get(UserStore);
+
+			const balanceChecks: { [key: string]: bigint } = {};
+
+			if (userStore.getKey(params.recipientAddress, tokenID).length === 0) {
+				if (params.accountInitializationFee !== USER_SUBSTORE_INITIALIZATION_FEE) {
+					throw new Error('Invalid account initialization fee');
+				}
+
+				balanceChecks[
+					TOKEN_ID_STORE_INITIALIZATION.toString('hex')
+				] = USER_SUBSTORE_INITIALIZATION_FEE;
+			}
+
+			const tokenIDSerialized = tokenID.toString('hex');
```

Comment:
```suggestion
			const hexTokenID = tokenID.toString('hex');
```
I think serialize means it's bytes, but in this case it's hex string